LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright Brightcove, Inc.
```

Comment:
We probably should apply [the new videojs governance and licensing](https://github.com/videojs/video.js/wiki/Governance#licensing-and-ownership) to all repos in the org. We should also make it more formal and actually put it into the videojs repo and what not.
Not sure exactly how this particular project should be copyrighted.